### PR TITLE
Adding TIA preview fields

### PR DESCRIPTION
**Pull request by kaadhina**

### CODE REVIEW

File: Tasks/VsTest/VSTest.ps1 [hunk 1]
```diff
@@ -14,7 +14,9 @@ param(
     [string]$publishRunAttachments,
     [string]$runInParallel,
     [string]$vstestLocationMethod,
-    [string]$vstestLocation
+    [string]$vstestLocation,
+    [string]$runOnlyImpactedTests,
```
Comment (acesiddhu) [inline Tasks/VsTest/VSTest.ps1]: lets write warning

File: Tasks/VsTest/vstest.ts [hunk 2]
```diff
@@ -34,9 +34,9 @@ try {
     var configuration: string = tl.getInput('configuration');
     var publishRunAttachments: string = tl.getInput('publishRunAttachments');
     var runInParallel: boolean = tl.getBoolInput('runInParallel');
-    var tiaEnabled = tl.getVariable('tia.enabled');
+    var tiaEnabled: boolean = tl.getBoolInput('runOnlyImpactedTests');
     var fileLevel = tl.getVariable('tia.filelevel');
```
Comment (acesiddhu) [inline Tasks/VsTest/vstest.ts]: hope file level is default option

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/VsTest/VSTest.ps1 (hunk 1 above)
Comment (kaadhina) [inline Tasks/VsTest/VSTest.ps1]: done

File: Tasks/VsTest/vstest.ts (hunk 2 above)
Comment (kaadhina) [inline Tasks/VsTest/vstest.ts]: yes

**Pull request merged**